_observing-your-data/alerting/monitors.md

```diff
@@ -18,11 +18,12 @@ redirect_from:
 
 ## Monitor types
 
-The OpenSearch Dashboard Alerting plugin provides four monitor types:
+The OpenSearch Dashboard Alerting plugin provides four basic monitor types, as well as a composite monitor that can utilize any or all monitor types:
```

Comment:
We only support three monitor types for composite monitor:
Doc level
Query level
Bucket level